lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -12,6 +12,8 @@
 namespace seahorn {
 namespace details {
 
+static const int slotBitWidth = 64;
```

Comment:
naming convention: `g_slotBitWidth` so that it is clear that this is a global value

Reply:
Done